doc/002-dev-workflow.md

```diff
@@ -10,11 +10,20 @@ Custom Hypersync apps are added to a Hyperproof organization using the [Hyperpro
 
 ### CLI Authorization
 
-After installing the CLI, you will first need to sign in using this command:
-
+After installing the CLI, you will first need to sign in using the 'hp signin' command:
 ```
 hp signin
 ```
+>  NOTE: For customers in the EU or GOV domain, the sign in command accepts an optional 'domain' parameter.
```

Comment:
`EU` and `GOV` are not domains.  In our code they are "offerings" but I don't think we say that in our docs.

Suggest you soften this.  e.g. "For customers using Hyperproof EU or Hyperproof Gov".  Those are both approved names for these services.